internal/storage/databases.go

```diff
@@ -0,0 +1,125 @@
+package storage
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	mariadbv1 "github.com/amazeeio/dbaas-operator/apis/mariadb/v1"
+	postgresv1 "github.com/amazeeio/dbaas-operator/apis/postgres/v1"
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	client "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
```

Comment:
You'll need to add this to ensure that the `role.yml` in the manifests for this are generated correctly, and run `make manifests` to update all the kustomize templates accordingly.

```
// +kubebuilder:rbac:groups=mariadb.amazee.io;postgres.amazee.io,resources=mariadbconsumers;postgresqlconsumers,verbs=list;watch
```

This is basically ensuring the same thing you've done in the [charts PR](https://github.com/uselagoon/lagoon-charts/pull/862/files#diff-454053b77f0224fcb121c778605f5438c8377c7b4766bbcbeb31d39f931aa021R77-R85) is stored in this repo too for local development

Reply:
Thanks, I didn't know what those comments were TIL.